Libraries/LibGUI/AbstractView.h

```diff
@@ -34,7 +34,7 @@ namespace GUI {
 
 class AbstractView : public ScrollableWidget {
     C_OBJECT_ABSTRACT(AbstractView)
-    friend class Model;
+    //friend class Model;
```

Comment:
Just nitpicking but in CONTRIBUTING.md it's advised not to include commented out code.

```suggestion
```

Reply:
Indeed, just remove it :)

Reply:
Oh! I thought I did that, but apparently forgot. Will do!